proteus/NumericalSolution.py

```diff
@@ -750,7 +751,38 @@ def PUMI2Proteus(self,mesh):
                         nFile=n,
                         analyticalSolution=p.analyticalSolution))
                 model.simTools = self.simOutputList[-1]
-                self.auxiliaryVariables[model.name]= [av.attachModel(model,self.ar[index]) for av in n.auxiliaryVariables]
+                
+                #Code to refresh attached gauges. The goal is to first purge 
+                #existing point gauge node associations as that may have changed
+                #If there is a line gauge, then all the points must be deleted
+                #and remade.
+                from collections import OrderedDict
+                for av in n.auxiliaryVariables:
+                  try:
+                    av.adapted=True
+                    for point, l_d in av.points.iteritems():
+                      if 'nearest_node' in l_d:
+                        l_d.pop('nearest_node')
+                    if(av.isLineGauge or av.isLineIntegralGauge): #if line gauges, need to remove all points
+                      av.points = OrderedDict()
+                    if(av.isGaugeOwner):
+                      for ptGaugeIdx,ptGaugeItem in enumerate(av.pointGaugeVecs):
+                        av.pointGaugeVecs[ptGaugeIdx].destroy();  
+                        av.pointGaugeMats[ptGaugeIdx].destroy();  
+                        av.dofsVecs[ptGaugeIdx].destroy();
+                      av.pointGaugeVecs = []
+                      av.pointGaugeMats = []
+                      av.dofsVecs = []
+                      av.field_ids=[]
+                      av.isGaugeOwner=False
+                  except AttributeError:
+                    pass
```

Comment:
I expect there to be a better way to do this but the reason the try/except structure is there is because there are auxiliary variables (AV) not related to gauges that to my knowledge don't need to be modified. For example, there is an AV associated with relaxation zones for some problems and they do not have a `isAdapted` variable in their class so they will trigger the `AttributeError` exception. So they are expected frequently.